lib/DxilContainer/DxilRDATBuilder.cpp

```diff
@@ -2,6 +2,7 @@
 #include "dxc/DxilContainer/DxilPipelineStateValidation.h"
 #include "dxc/Support/FileIOHelper.h"
 #include "dxc/Support/Global.h"
+#include <assert.h> // Needed for DxilPipelineStateValidation.h
```

Comment:
clang-format autopublish moved this assert here, but it needs to be included before `DxilPipelineStateValidation.h`.  So I believe it needs to be separated by a blank line, right?

Reply:
@tex3d Yes, you're right. Added a blank line to fix it.